uberenv.py

```diff
@@ -464,10 +594,9 @@ def install(self):
         install_cmd = "spack/bin/spack "
         if self.opts["ignore_ssl_errors"]:
             install_cmd += "-k "
-        if not self.opts["install"]:
-            install_cmd += "dev-build -d {} -u {} ".format(self.pkg_src_dir,self.pkg_final_phase)
-        else:
-            install_cmd += "install "
+        install_cmd += "dev-build -d {0} ".format(self.pkg_src_dir)
+        if not self.opts["install"] and self.pkg_final_phase:
+            install_cmd += "-u {0} ".format(self.pkg_final_phase)
             if self.opts["run_tests"]:
```

Comment:
While i was reverting some work, I noticed this run test option was not in the "not install" if.  Should this be deindented so it hits both cases?

Reply:
Thanks for noticing this, now I remember!
So, in #20 I noticed that there were no more testing support because dev-build does not support tests. That’s why in re-introduced the `install` command.
This also explain why we cannot "deindent" this.
Maybe we could ask the Spack guys to add support for tests in the dev-build command as-well.